packages/vuetify/src/components/VTabs/VTab.sass

```diff
@@ -2,8 +2,8 @@
 
 .v-tab
   // override v-btn size specificity
-  &.v-tab
-    --v-btn-height: var(--v-tabs-height)
+  &.v-tab.v-btn
```

Comment:
had to increase .v-tab.v-tab selector specificity to override height from this one:
```css
.v-btn.v-btn--density-default {
    height: calc(var(--v-btn-height) + 0px);
}